### DISCUSSION TITLE
Is there an intruduction example that shows Popupmenues?

### DISCUSSION THREAD

**Discussion by thomasstuempfig** [maxGraph/maxGraph#584] [Q&A] [answered]
Hi all,
i am trying to get maxgraph working in an vue3 environnement.
I successfully can generate a graph insert edges and vertices.
Now I'd like to add a Popupmenu. The Popupmenue appears not at/near the mouse position. 
I was not able to use HandleConfig as in the example and I suppose this could be the problem. How should I use HandleConfig in the code.
The documentation lacks complete examples. I cannot import HandleConfig.

regards
Thomas
Thomas

**Comment by HalilFocic** [comment, marked as answer]
Hi @thomasstuempfig ,
Did you check this example? https://maxgraph.github.io/maxGraph/demo/?path=/story/misc-menustyle--default ? 
I am assuming you want behaviour like this. 
You can also style this popup menu by targeting the classes using css.
HandleConfig is imported from @maxgraph/core like this:
![image](https://github.com/user-attachments/assets/d0923a45-59b1-4be7-8815-56f20b5e7da3)


Hope it helps. If not, let me know how I can assist

**Reply by thomasstuempfig** [reply]
first , thank you for the quick reply.
I imported it like you did
e.g
import { Graph, InternalEvent,PopupMenuHandler,CellRenderer,CellEditorHandler , HandleConfig, TooltipHandler, SelectionCellsHandler, ConnectionHandler, SelectionHandler, PanningHandler, RubberBandHandler} from '@maxgraph/core';


![grafik](https://github.com/user-attachments/assets/52c8b2e2-24ca-460a-aac4-b6f629064059)
i get the Error above

**Reply by thomasstuempfig** [reply]
Additional hint: After a search  for all files containing HandleConfig I did not find any file in node_modules.

**Reply by tbouffard** [reply]
Hi @thomasstuempfig, I guess you are using maxGraph 0.13.1. 
The stories in the main branch are using the code of maxGraph from the main branch as well, so they can relied on unreleased code.

`HandleConfig` is available as of version 0.14.0 which is not released yet. It doesn't relate to PopupMenu so I suggest you not to import it in your code.
You can also have a look at https://maxgraph.github.io/maxGraph/demo/?path=/story/misc-showregion--default
The code of the stories is available at https://github.com/maxGraph/maxGraph/blob/main/packages/html/stories/ShowRegion.stories.ts

**Reply by thomasstuempfig** [reply]
Hi @tbouffard 
Thank you for the hint. I will check it out.
regards
Thomas

**Comment by thomasstuempfig** [comment]
Hi @tbouffard 
problem solved. Indeed it was the css that was missing. 

regards
Thomas

**Reply by tbouffard** [reply]
You are welcome.
Authorship of the solution goes to @HalilFocic who mentioned the potential CSS problem 😄
